src/lib.rs

```diff
@@ -10,11 +10,9 @@ extern crate std;
 
 pub mod batch;
 mod error;
-mod signature;
 mod signing_key;
 mod verification_key;
 
 pub use error::Error;
-pub use signature::Signature;
```

Comment:
Sorry, one last thing I noticed :sweat: . Can you please reexport Signature here, so that users of the library don't need to import the `ed25519` crate? e.g.

```
pub use ed25519::Signature
```